ansible-ipi-install/roles/node-prep/vars/main.yml

```diff
@@ -1,7 +1,8 @@
 ---
 # vars file for node-prep
 package_list:
-  - firewalld
+  - "{{ firewall }}"
```

Comment:
This one is interesting because right now firewall can only be firewalld or iptables. However, the logic in the 10_validation.yml is looking to see if iptables-service exists or not, if it does, we know you explicitly wanted to use iptables. In its current form it doesn't break anything because by default it uses firewalld and when iptables-service is installed, this variable changes to iptables but you likely already have iptables installed on the system since you had iptables-service. I just wanted to point that out since this won't install iptables-service which we only ever want if we actually going to use iptables.

Reply:
My argument for leaving this here is that if you have iptables in lieu of firewalld, you probably don't want the firewalld package to get installed. This will prevent that.